src/index.html

```diff
@@ -6,9 +6,65 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap"
+          rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <!--    LOGO:        -->
+      <a href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="LOGO"
+          width="40"
+          height="40"
+        >
+      </a>
+
+
+      <!--     NAV BAR:         -->
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a class="nav__list__link is-active" href="#">APPLE</a>
```

Comment:
Use text-transform

Reply:
Use text-transform

Reply:
@OleksiiBlezniuk I used text-transform: uppercase but in css, is it wrong?(

Reply:
No, it's right, but there is no need to make this text uppercased